### DISCUSSION TITLE
No URL for autorization

### DISCUSSION THREAD

**Discussion by goomis90** [ho-dev/HattrickOrganizer#1875] [Q&A]
How can I generate URL for autorization with Hattrick?
There isn't any URL in autorization window and "Open URL" button do nothing :(

![image](https://github.com/akasolace/HO/assets/91156530/67f33493-cc79-4fad-ba96-26e703ed2f0a)
![image](https://github.com/akasolace/HO/assets/91156530/5650e0e2-88f2-4d57-b3d3-3d408c9e4720)

**Comment by KankisLTU** [comment]
Same for me.

**Comment by KankisLTU** [comment]
OK, found a solution to that, just make inbound and outbound rules for HO and it's gonna work

**Reply by KenniZz** [reply]
Can you please explain what you mean with make in and outbound rules?
